README.md

```diff
@@ -559,8 +594,9 @@ while true; do
             case "${options[$selected]}" in
                 "Audio Server")
                     audio_server
-                    echo -e "You can login as CLI now.\nSelect 'Login CLI' to login!\n"
+                    echo -e "\nYou can login as CLI now.\nSelect 'Login CLI' to login!\n"
                     wait_for_key_press
+                    $HOME/chroot-xfce.sh
                     ;;
                 "Login CLI")
                     su -c mount -o remount,dev,suid /data
```

Comment:
There should be no need to remount the `/data` folder every time you login to distribution, it is required to do only once. There could be side-effects by doing it every time. Also, suid should be enough as there is should be no need for dev as `/dev` will be bind mounted. The same applies to to Login GUI.